packages/icons/tests/chromatic/Icons.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import type { ContextValue } from "react-aria-components";
+import type { ResponsiveProp, ColorValue } from "@hopper-ui/styled-system";
+import * as IconLibrary from "@hopper-ui/icons";
+
+import "./icons.stories.css";
+
+interface IconLibraryType {
+    [key: string]: React.ComponentType<ContextValue<Omit<IconLibrary.IconProps, "src16" | "src24" | "src32" >, SVGSVGElement>>;
+}
+
+interface ListProps {
+    icons: string[];
```

Comment:
i think you can skip the icons parameters and use IconLibrary.iconNames directly